sdk/communication/azure-communication-chat/build.gradle

```diff
@@ -34,24 +34,24 @@ static def getBuildVariant() {
 
```

Comment:
Update Chat version

Reply:
This PR is not for preparing Chat for the GA release. I think we should update the version in a separate PR.

Reply:
Sorry, I thought it was. If we are expecting another PR for releasing, sure. Just ignore me.